tests/functional/sign_in.js

```diff
@@ -113,7 +113,7 @@ define([
       var email = TestHelpers.createEmail();
       return fillOutSignIn(this, email, PASSWORD)
         // The error area shows a link to /signup
-        .then(FunctionalHelpers.visibleByQSA('.error a[href="/signup"]'))
+        .then(FunctionalHelpers.visibleByQSAHeight('.error a[href="/signup"]', undefined, 10))
```

Comment:
@vladikoff Noticed that OAuth uses https://github.com/mozilla/fxa-content-server/blob/51cdfc3ccf402d7830ed612872dd7184deb31896/tests/functional/oauth_sign_up.js#L230. 

Which looks like it is testing for an attribute on the element to determine if it shown, then removes it. Is it possible to replace `testErrorWasShown` with `visibleByQSA`? Or do these serve different purposes?